README.md

```diff
@@ -58,5 +58,16 @@ This can be verified in the tests; tests will not pass with checkstyle errors.
 ## Releases
 The `aws-advanced-wrapper-driver` has a regular monthly release cadence. A new release will occur during the last week of each month. However, if there are no changes since the latest release, then a release will not occur.
```

Comment:
Outside of this review, but could you fix this?
```suggestion
The `aws-advanced-jdbc-wrapper` has a regular monthly release cadence. A new release will occur during the last week of each month. However, if there are no changes since the latest release, then a release will not occur.
```